### ISSUE TITLE
Trac bug crawl explodes

### ISSUE THREAD

**Issue by imported-from-roundup** [openhatch/oh-mainline#480]
<b>Comment by <a href='https://openhatch.org/people/paulproteus/'>paulproteus</a>:</b><dl><dd>2010-10-17 00:18:44,826 update:208 INFO     Started refreshing all Trac bugs.
Traceback (most recent call last):
  File "./bin/production", line 78, in &lt;module&gt;
    djangorecipe.manage.main('mysite.deployment_settings')
  File
"/home/deploy/milestone-a/eggs/djangorecipe-0.20-py2.6.egg/djangorecipe/manage.py",
line 16, in main
    management.execute_manager(mod)
  File
"/home/deploy/milestone-a/parts/production/django/core/management/**init**.py",
line 362, in execute_manager
    utility.execute()
  File
"/home/deploy/milestone-a/parts/production/django/core/management/**init**.py",
line 303, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File
"/home/deploy/milestone-a/parts/production/django/core/management/base.py", line
195, in run_from_argv
    self.execute(_args, *_options.**dict**)
  File
"/home/deploy/milestone-a/parts/production/django/core/management/base.py", line
222, in execute
    output = self.handle(_args, *_options)
  File
"/home/deploy/milestone-a/mysite/customs/management/commands/customs_daily_tasks.py",
line 156, in handle
    self.find_and_update_enabled_trac_instances()
  File
"/home/deploy/milestone-a/mysite/customs/management/commands/customs_daily_tasks.py",
line 84, in find_and_update_enabled_trac_instances
    instantiated.update()
  File "/home/deploy/milestone-a/mysite/customs/bugtrackers/trac.py", line 213,
in update
    for bug_id in self.generate_list_of_bug_ids_to_look_at():
  File "/home/deploy/milestone-a/mysite/customs/bugtrackers/trac.py", line 199,
in generate_bug_ids_from_queries
    mysite.customs.bugtrackers.trac.csv_of_bugs(query_url))
  File "/home/deploy/milestone-a/mysite/customs/bugtrackers/trac.py", line 27,
in csv_url2list_of_bug_ids
    return [int(line['id']) for line in dictreader]
KeyError: 'id'

</dl></dd><hr/>

<b>Comment by <a href='https://openhatch.org/people/paulproteus/'>paulproteus</a>:</b><dl><dd>It should be easy to write a quick fix for this, but I haven't yet.

</dl></dd><hr/>

<b>Comment by <a href='https://openhatch.org/people/paulproteus/'>paulproteus</a>:</b><dl><dd>The problem is fantastic.

Long story short, we're suffering from <a href="http://bugs.python.org/issue7185">http://bugs.python.org/issue7185</a>

Fixed on master; deploying!

</dl></dd><hr/>

Status: <b>resolved</b>
Nosy List: <b>  paulproteus, pythonian4000 </b>
Priority: <b>critical</b>
Imported from roundup ID: <b>165</b> (<a href='http://roundup-archive.openhatch.org/bugs/issue165'>view archived page</a>)
Last modified: <b>2010-12-14.03:39:52</b>
